velox/dwio/parquet/CMakeLists.txt

```diff
@@ -28,8 +27,8 @@ add_library(velox_dwio_parquet_writer RegisterParquetWriter.cpp)
 
 if(VELOX_ENABLE_PARQUET)
   target_link_libraries(
-    velox_dwio_parquet_reader velox_dwio_duckdb_parquet_reader
-    velox_dwio_native_parquet_reader xsimd)
+    velox_dwio_parquet_reader velox_dwio_native_parquet_reader
+    velox_dwio_arrow_parquet_writer xsimd)
```

Comment:
Why does the native parquet reader require the dwio_arrow_parquet_writer ?

Reply:
I think this was a mistake, I removed `dwio_arrow_parquet_writer`.